extensions/wikia/AdEngine/js/template/bfaa.js

```diff
@@ -100,6 +110,25 @@ define('ext.wikia.adEngine.template.bfaa', [
 		}
 	};
 
+	function isRecoveryNotSupportedBrowser() {
+		return browser.isIE() || browser.isEdge();
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this commented out code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=javascript%3ACommentedCode)